frontend/src/components/shared/InputInstructions/InputInstructions.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { ReactElement } from "react"
+import { isFromMac } from "lib/utils"
+import classNames from "classnames"
+
+import "./style.scss"
+
+export interface Props {
+  dirty: boolean
+  value: string
+  maxLength?: number
+  className?: string
+  type?: "multiline" | "single"
+}
+
+const InputInstructions = ({
+  dirty,
+  value,
+  maxLength,
+  className,
+  type = "single",
+}: Props): ReactElement => {
+  const containerClassName = classNames("instructions", className)
+  let message
+
+  if (type === "multiline") {
+    if (isFromMac()) {
+      message = "Press ⌘+Enter to apply"
+    }
+
+    if (!isFromMac()) {
+      message = "Press Ctrl+Enter to apply"
+    }
+  } else {
+    message = "Press Enter to apply"
+  }
+
+  if (dirty && maxLength && value.length > 0) {
```

Comment:
I think the logic is a little off. We should show always the fraction "x/y" when there's a `maxLength`.

So the code should be more like:

```
messages = []

if (dirty) {
  messages.push(
    <span className="message">{message}</span>
  )
}

if (maxLength != null) {
  messages.push(
    <span
      className={classNames("counter", {
        blink: value.length >= maxLength,
      })}
    >
      {value.length}/{maxLength}
    </span>
  )
}

return (
  <div ...>
    { messages }
  </div>
)
```